### DISCUSSION TITLE
Noob Need Help Opening Program (Please Help!)

### DISCUSSION THREAD

**Discussion by leadsteelcopper** [AAndyProgram/SCrawler#91] [Q&A] [answered]
Hello,

I am a noob when it comes to software and this is the first time I am using GitHub so sorry if this post is quite basic. When I download the zip file and use the extract option in Windows is places all the files in a new folder. When double clicking the ffmeg application the screen flickers black for a moment and then nothing happens. Am I doing something wrong? Please help if you can, and sorry ahead of time if this is too noobish.

Thank you AndyProgram for providing and updating this program,

**Comment by AAndyProgram** [comment, marked as answer]
Why are you clicking on fffmpeg.exe? You need to click on SCrawler.exe

**Reply by leadsteelcopper** [reply]
Oh it worked. Sorry I was not paying attention yesterday, I knew it was something dumb. Thank you!

**Reply by AAndyProgram** [reply]
You are welcome 😉
